pom.xml

```diff
@@ -70,9 +70,6 @@
         <module>config</module>
         <module>all</module>
         <module>examples</module>
-        <module>extension/contentfragment/bundle</module>
-        <module>extension/contentfragment/content</module>
-        <module>extension</module>
```

Comment:
I would still keep the extension folder in case we add stuff in it in the future.

Reply:
Keeping the folder or the module? The latter would mean maintaining the pom + generating an empty content-package (which is only possible by setting the plugin config to `failOnEmptyFilter=false`). The first also doesn't make much sense IMHO. WDYT @jckautzmann @vladbailescu ?

Reply:
+1 for removing both folder and module, if they don't serve a purpose. We can always reintroduce them if there's some need later.

Reply:
I'm fine if we remove both folder and module.
@philipmundt Could you please verify that upgrading to the new version of the Core Components does not break existing implementations based on the `core/wcm/extension/components/contentfragment/v1/contentfragment` resource type.